requirements_all_ds.txt

```diff
@@ -32,3 +32,5 @@ phoenixdb==0.7
 certifi>=2019.9.11
 pydgraph==2.0.2
 azure-kusto-data==0.0.35
+pyexasol==0.9.1
+python-rapidjson==0.8.0
```

Comment:
Did you notice any noticeable performance difference when using this?

Reply:
Its recommended in the best practices of pyexasol, i have not done any benchmarking on this by myself. 

https://github.com/exasol/pyexasol/blob/master/docs/BEST_PRACTICES.md#consider-faster-json-parsing-libraries